jera customization

mohammedabida opened the pull request.

mohammedabida committed:
jera customization

jera customization
update sdcp-be root ulr in components
fix sdcp-be token in components
fix passing sdcp-be apikey into request headers
jera_customization: UI changes (Store button removed, GitHub Button removed, Jera Icon added, New project Functionality changed to blank workflow)

autofix-troubleshooter commented:
Hi! I'm the <a href="https://autofix.ci"><img alt="autofix logo" src="https://autofix.ci/logo/logo.svg" width="16"><strong>autofix.ci</strong></a> troubleshooter bot.

It looks like you correctly set up a CI job that uses the autofix.ci GitHub Action, but the [**autofix.ci GitHub App**](https://github.com/apps/autofix-ci/) has not been installed for this repository. This means that autofix.ci unfortunately does not have the permissions to fix this pull request. If you are the repository owner, please install the app and then restart the CI workflow! 😃

yassminesassi committed:
164906- API for sharing flow

164906 - fix flows_share_router import issues
164906 - fxing package-lok.json
164906 - added changes according to skander latest comments
164906 - added user verification change db schema
164906 - working API for sharing flow
